README.md

```diff
@@ -1,5 +1,4 @@
-Twirl
-=====
+# Twirl [![Latest version](https://index.scala-lang.org/playframework/twirl/twirl-api/latest.svg?color=orange)](https://index.scala-lang.org/playframework/twirl/twirl-api)
```

Comment:
You can use this version because it is consistent across all twirl components here.